### PR TITLE
App 1174 subgraph adapt add contract events registries

### PR DESCRIPTION
**Pull request by mathewmeconry**
## Description

Extends subgraph with more events.

Task: APP-1174

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [x] I have updated the ``CHANGELOG.md`` file in the root folder.
- [ ] I have tested my code on the test network.

### CODE REVIEW

File: packages/subgraph/schema.graphql [hunk 1]
```diff
@@ -236,3 +237,46 @@ type AllowlistProposal implements Proposal @entity {
...
+  dao: Dao!
+  pluginSetup: PluginSetup!
+  data: Bytes!
+  helpers: [PluginHelper!]! @derivedFrom(field: "plugin")
+  state: PluginState!
+  sender: String!
+}
+
+enum PluginState {
+  InstallationPending
```
Comment (Michael-A-Heuer) [inline packages/subgraph/schema.graphql:277]: Is `XYZPending` a good name? In `PluginSetupProcessor` we call it `XYZPrepared`

**Review by Michael-A-Heuer — COMMENTED** [review]
Minor suggestions regarding event names and NatSpec

File: packages/subgraph/manifest/subgraph.placeholder.yaml [hunk 2]
```diff
@@ -122,3 +174,22 @@ templates:
...
+      language: wasm/assemblyscript
+      file: ./src/plugin/pluginRepo.ts
+      entities:
+        - PluginRepo
+      abis:
+        - name: PluginRepo
+          file: $ZARAGOZA_CONTRACTS_MODULE/artifacts/contracts/plugin/PluginRepo.sol/PluginRepo.json
+      eventHandlers:
+        - event: VersionCreated(uint256,uint16[3],indexed address,bytes)
+          handler: handleVersionCreated
```
Comment (Rekard0) [inline packages/subgraph/manifest/subgraph.placeholder.yaml:195]: add new empty line at the end

File: packages/subgraph/manifest/data/rinkeby.json [hunk 3]
```diff
@@ -6,6 +6,16 @@
...
+    },
+    "PluginRepoRegistry": {
+      "name": "PluginRepoRegistry",
+      "address": "0x2B8C4DD137104d1E869105cd0106e7D9EF955BfE",
+      "startBlock": 7727664
+    },
+    "PluginSetupProcessor": {
+      "name": "PluginRepoRegistry",
+      "address": "0x2B8C4DD137104d1E869105cd0106e7D9EF955BfE",
+      "startBlock": 7727664
```
Comment (Rekard0) [inline packages/subgraph/manifest/data/rinkeby.json:18]: when do u think we should remove rinkeby ?

**Review by Rekard0 — APPROVED** [review]
LGTM

File: packages/subgraph/schema.graphql (hunk 1 above)
Comment (mathewmeconry) [reply packages/subgraph/schema.graphql]: good point. done

File: packages/contracts/contracts/plugin/PluginRepo.sol [hunk 4]
```diff
@@ -75,7 +75,7 @@ contract PluginRepo is
     /// @notice Emitted when a new version is created.
     /// @param versionId The version index.
     /// @param semanticVersion The semantic version number.
-    event VersionCreated(uint256 versionId, uint16[3] semanticVersion);
+    event VersionCreated(uint256 versionId, uint16[3] semanticVersion, address indexed pluginSetup, bytes contentURI);
```
[the remark this answers is not in the document]
Comment (mathewmeconry) [reply packages/contracts/contracts/plugin/PluginRepo.sol:80]: good point. done

File: packages/subgraph/manifest/data/rinkeby.json (hunk 3 above)
Comment (mathewmeconry) [reply packages/subgraph/manifest/data/rinkeby.json]: now xD

**Pull request merged**